lib/rules/no-native-reassign.js

```diff
@@ -55,7 +55,7 @@ module.exports = {
             ) {
                 context.report({
                     node: identifier,
-                    message: "{{name}} is a read-only native object.",
+                    message: "Unexpected modification of read-only global variable '{{name}}'.",
```

Comment:
I'm not a native English speaker, and I have often seen the word "Unexpected" in parser errors.
It's the reason.

> "Unexpected" implies that there's a valid "expected" way to change the read-only global

I learned it today.
Thank you!